grails-app/services/org/pih/warehouse/product/ProductService.groovy

```diff
@@ -1228,15 +1228,40 @@ class ProductService {
         return productInstance
     }
 
+    Attribute findExternalProductIdAttribute(String externalAttributeCode = null){
+        if (!externalAttributeCode) {
+            externalAttributeCode = grailsApplication.config.externalProductId.attribute.code
+        }
+        if (!externalAttributeCode) {
+            externalAttributeCode = "EXTERNAL_PRODUCT_ID"
+        }
```

Comment:
Remove 1235-1237. If we did want to do something like this in the code it should 
```
externalAttributeCode = grailsApplication.config.openboxes.product.externalProductId.attributeCode?:"EXTERNAL_PRODUCT_ID"
```
But we don't want that now since we'll have it configured in the Config.groovy or openboxes-config.properties or openboxes-config.groovy